azure/table_azure_ad_service_principal.go

```diff
@@ -120,98 +115,13 @@ func tableAzureAdServicePrincipal(_ context.Context) *plugin.Table {
 				Description: "A list of service principal names.",
 				Type:        proto.ColumnType_JSON,
 			},
-
-			// Steampipe standard columns
-			{
-				Name:        "title",
-				Description: ColumnDescriptionTitle,
-				Type:        proto.ColumnType_STRING,
-				Transform:   transform.FromP(getAdServicePrincipalTurbotData, "TurbotTitle"),
-			},
-			{
-				Name:        "akas",
-				Description: ColumnDescriptionAkas,
-				Type:        proto.ColumnType_JSON,
-				Transform:   transform.FromP(getAdServicePrincipalTurbotData, "TurbotAkas"),
-			},
 		},
 	}
 }
 
 //// FETCH FUNCTIONS
 
 func listAdServicePrincipals(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
-	session, err := GetNewSession(ctx, d, "GRAPH")
-	if err != nil {
-		return nil, err
-	}
-	tenantID := session.TenantID
-
-	graphClient := graphrbac.NewServicePrincipalsClientWithBaseURI(session.GraphEndpoint, tenantID)
-	graphClient.Authorizer = session.Authorizer
-
-	result, err := graphClient.List(ctx, "")
-	if err != nil {
-		return nil, err
-	}
-	for _, servicePrincipal := range result.Values() {
-		d.StreamListItem(ctx, servicePrincipal)
-	}
-
-	for result.NotDone() {
-		err = result.NextWithContext(ctx)
-		if err != nil {
-			return nil, err
-		}
-
-		for _, servicePrincipal := range result.Values() {
-			d.StreamListItem(ctx, servicePrincipal)
-		}
-	}
-
+	err := errors.New("Table deprecated and remove, please use table azuread_service_principal instead.")
```

Comment:
```suggestion
	err := errors.New("Table has been deprecated and removed, please use table azuread_service_principal instead.")
```

My original message looked a bit off, can you please try this message and see how it appears when querying this table?